ambry-router/src/test/java/com.github.ambry.router/SimpleOperationTrackerTest.java

```diff
@@ -388,6 +390,63 @@ public void useReplicaNotSucceededSendTest() {
   }
 
   /**
+   * Test to ensure that replicas that are down are also returned by the operation tracker, but they are only
```

Comment:
Just don't think the word "attempt" fits in anywhere here at the operation tracker.  Just like your description says, attempting to send requests to these down replicas is up to the `Router`. The operation tracker just orders them.